userspace/libscap/engine/modern_bpf/scap_modern_bpf.c

```diff
@@ -62,6 +67,126 @@ static int32_t populate_64bit_interesting_syscalls_table(bool* ppm_sc_array)
 	return ret;
 }
 
+static bool read_next_kernel_config_option(gzFile file, char *buf, size_t n,
+					   char **value)
+{
+	char *sep;
+
+	while (gzgets(file, buf, n)) {
```

Comment:
Are we only expecting compressed? Could we support both options?

Reply:
Yes, both ways are supported -- `gzopen` works with uncompressed files as well, falling back to a simple read [1]. In fact I was testing it on an uncompressed config.

[1]: https://www.zlib.net/manual.html